### PR TITLE
Disable metadata plugin if manifest doesn't have metadata

### PR DESCRIPTION
**Pull request by EricHanLiu**
Fixes issue #425

### CODE REVIEW

**Comment by EricHanLiu** [comment]
Forgot to update the css file that karma includes, fixes unit tests failing

**Comment by EricHanLiu** [comment]
Odd, works locally with headless but fails in Travis, it's likely because of the css file but shouldn't be an issue. Will look into it tomorrow

File: source/js/plugins/metadata.js [hunk 1]
```diff
@@ -34,21 +36,25 @@ export default class MetadataPlugin
      * Display a modal with the IIIF manifest metadata. Here, viewer refers to the Diva instance (because
      * the toolbar class refers to it as viewer)
      **/
-    handleClick (viewer) 
+    handleClick () 
     {
         // if first click create div elements
         let metadataDiv;
+
+        let metadata = this.core.viewerState.manifest.metadata;
```
Comment (ahankinson) [inline source/js/plugins/metadata.js]: Do you need to handle the absence of metadata here? If the toolbar icon is not created, then the click handler is never attached, thus this method will never get called.

File: source/js/plugins/metadata.js [hunk 2]
```diff
@@ -34,21 +36,25 @@ export default class MetadataPlugin
      * Display a modal with the IIIF manifest metadata. Here, viewer refers to the Diva instance (because
```
Comment (ahankinson) [inline source/js/plugins/metadata.js]: You need to update this comment

Comment (ahankinson) [inline source/js/plugins/metadata.js]: You need to update this comment

File: source/js/plugins/metadata.js [hunk 3]
```diff
@@ -34,21 +36,25 @@ export default class MetadataPlugin
...
      * the toolbar class refers to it as viewer)
      **/
-    handleClick (viewer) 
+    handleClick () 
     {
         // if first click create div elements
         let metadataDiv;
+
```
Comment (ahankinson) [inline source/js/plugins/metadata.js]: Do you need to handle the absence of metadata here? If the toolbar icon is not created, then the click handler is never attached, thus this method will never get called.

File: source/js/diva.js [hunk 4]
```diff
@@ -123,7 +120,11 @@ class Diva
             toolbar: this.settings.enableToolbar ? this.toolbar : null
         };
 
-        this.toolbar.render();
+        // only render the toolbar after the object has been loaded
+        Diva.Events.subscribe('ObjectDidLoad', () =>
+        {
+            this.toolbar.render();
```
Comment (ahankinson) [inline source/js/diva.js]: 👍 🎉 🎆

**Comment by ahankinson** [comment]
You probably need to run a build of Diva prior to running the tests. I would suggest running `npm run build:production` so that we're testing the production builds.

**Pull request merged**